src/goInstallTools.ts

```diff
@@ -383,8 +383,25 @@ export function checkLanguageServer(): boolean {
 	if (!langServerAvailable) {
 		promptForMissingTool('go-langserver');
 		vscode.window.showInformationMessage('Reload VS Code window after installing the Go language server');
+		return langServerAvailable;
 	}
-	return langServerAvailable;
+
+	// we execute the languageserver using -help so that it will fail and
```

Comment:
This is pretty terrible, but it's the best I could come up with in case we want to make sure an up-to-date version of the language server is installed.

Alternatively we could call go-langserver -version, but I don't really know what do to with the output as it would currently just print sth like v2-dev: https://github.com/sourcegraph/go-langserver/blob/dd4c619b53ae9d18c94e372d23d8710e59f0766b/main.go#L41